Digipost/APIClient.swift

```diff
@@ -445,6 +452,8 @@ class APIClient : NSObject, NSURLSessionTaskDelegate, NSURLSessionDelegate, NSUR
                 }, failure: { (error) -> Void in
                     if error.code == Int(SHCOAuthErrorCode.InvalidRefreshTokenResponse.rawValue) {
                         self.deleteRefreshTokensAndLogoutUser()
+                    } else {
+                        Logger.dpostLogError("Deleting oauth tokens and logout user. Not invalid refresh token.  error: \(error)")
```

Comment:
Don't log the full error object, be more specific, so we don't accidentally log something we don't want to